app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
Why you use BaseException here and Exception for VaccineError? What different between them?